packages/account-center/src/pages/Home/index.tsx

```diff
@@ -0,0 +1,7 @@
+import ErrorPage from '@ac/components/ErrorPage';
+
+const Home = () => (
+  <ErrorPage titleKey="account_center.home.title" messageKey="account_center.home.description" />
+);
+
+export default Home;
```

Comment:
Only for testing?

Reply:
The homepage will be implemented in the future, only sub-pages for phrase 1